composer.json

```diff
@@ -20,7 +20,7 @@
     "GPL-2.0+"
   ],
   "require": {
-    "typo3/cms-core": ">=7.6.0,<8.9.99"
+    "typo3/cms": ">=7.6.13,<8.9.99"
```

Comment:
Sounds fair enough to me - although it isn't technically necessary. Would you prefer that as a second commit we can merge first, then remove the composer change part of this PR?

On the subject of `-core` or not, I will also suggest to Helmut to use the `typo3-cms` name in `typo3_console` as well. Not to be disruptive, I just think it's the more correct way (given the current state of the TYPO3 composer manifest which `replaces` the `-core` name).